scan/peers.py

```diff
@@ -27,6 +27,10 @@
 
 logger = logging.getLogger(__name__)
 
+if PEERS_SCAN_DELAY > 0:
```

Comment:
This should actually be removed all along, makes no sense to have it inside the program. The scheduler running this that should take care of it.

Reply:
Probably true.  It's designed to work with supervisord though which restarts the script if it ever stops running.  So without the sleep supervisor runs it in a loop back to back.   Value can be zero for those that don't use supervisord.

Reply:
Good, let's leave as is.